tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fxaccounts.py

```diff
@@ -39,13 +50,16 @@ def enter_email(self, email=None):
         Wait(self.marionette).until(lambda m: self.keyboard.is_keyboard_displayed)
         self._switch_to_fxa_iframe()
         self.marionette.find_element(*self._next_locator).tap()
+        passwd_page = Wait(self.marionette, timeout=60).until(
+            expected.element_present(*self._password_page_locator))
+        Wait(self.marionette).until(expected.element_displayed(passwd_page))
```

Comment:
actually, sometimes it's in a state where the element is present, but not yet fully displayed.